pages.fr/common/tar.md

```diff
@@ -0,0 +1,33 @@
+# tar
+
+> Utilitaire d'archivage.
+> Souvent combiné avec une méthode de compression, tel que gzip ou bzip.
+> Plus d'informations : <https://www.gnu.org/software/tar>.
+
+- Créer une archive à partir de fichiers :
+
+`tar cf {{cible.tar}} {{fichier1}} {{fichier2}} {{fichier3}}`
```

Comment:
please, don't change the command format. the original one is:
`tar cf {{target.tar}} {{file1 file2 file3}}`, so I think it is better to keep `{{file1 file2 file3}}` format into your translation:
```suggestion
`tar cf {{cible.tar}} {{fichier1 fichier2 fichier3}}`
```